Mithril update 2024-07-31 (#438)
IntersectMBO/cardano-updates: blog/2024-07-31-mithril.md

---
title: Mithril Team Update
slug: 2024-07-31-mithril
authors: jpraynaud
tags: [mithril]
hide_table_of_contents: false
---

## High level overview

> This week, the Mithril team released the new distribution [`2430.0`](https://github.com/input-output-hk/mithril/releases/tag/2430.0) which includes support for Cardano node version `9.1.0`, activation of the Cardano transactions certification in `release-preprod` and `pre-release-preview` networks, as well as bug fixes and performance improvements. They kept working on the certification of Cardano stake distribution, activated the certification in the tests networks, and started working on the implementation of the new routes of the aggregator REST API. 

Reviewer: ```suggestion
This week, the Mithril team released the new distribution [`2430.0`](https://github.com/input-output-hk/mithril/releases/tag/2430.0) which includes support for Cardano node version `9.1.0`. They activated the certification of Cardano transactions in the `release-preprod` and `pre-release-preview` networks, and also included bug fixes and performance improvements. The team continued their work on the certification of Cardano stake distribution, activated the certification in the test networks, and started working on implementing new routes for the aggregator REST API.. 
```

> Finally, the team kept benchmarking the footprint of the signer on the SPO infrastructure and thoroughly tested edge cases for the Cardano transaction certification, and continued redacting a CIP for the diffusion of Mithril signatures through the Cardano network.

Reviewer: ```suggestion
Finally, the team benchmarked the signer's footprint on the SPO infrastructure, thoroughly tested edge cases for Cardano transaction certification, and continued drafting a CIP for the diffusion of Mithril signatures through the Cardano network.
```
